CHANGELOG.rst

```diff
@@ -3,19 +3,22 @@
 0.52.0
 ------
 
-Renku ``0.52.0`` contains a new secrets-storage service that allows users to store and use secrets in 
-sessions.
-
+Renku ``0.52.0`` introduces new features related to Renku 2.0. However, Renku 2.0 is still
+in early development and is not yet accessible to users. For more information, see our
+[roadmap](https://github.com/SwissDataScienceCenter/renku-design-docs/blob/main/roadmap.md).
```

Comment:
```suggestion
Renku ``0.52.0`` introduces a new secrets-storage service that allows users to store and use secrets in 
sessions.

This release also contains new features related to Renku 2.0. However, Renku 2.0 is still
in early development and is not yet accessible to users. For more information, see our
[blog](https://blog.renkulab.io/renku-2).
```

Reply:
I had to resolve conflicts locally so I just copied and pasted this.